game-backend/src/server.ts

```diff
@@ -7,9 +7,15 @@ const server = http.createServer();
 const WSServer = new WebSocket.Server({
   server,
 });
+const mainController = new MainController();
 
-WSServer.on('connection', ws => {
-  MainController.newConnection(ws);
+WSServer.on('connection', (ws: WebSocket, req: http.IncomingMessage) => {
+  const connectionId = req.headers['sec-websocket-key'] as string;
+  mainController.newConnection(ws, connectionId);
+
+  ws.on('message', msg => {
+    MainController.newMessage(msg, ws, connectionId);
+  });
```

Comment:
@sir-marc Should we move this change also out of this branch? This would enable us to have only the tick engine itself.

Reply:
I would like to move it.

Reply:
Fine for me, added this to "prove" the POC ^^ and because i thought, we will add the command sending already. But can be removed.